llvm/lib/Target/AMDGPU/SIRegisterInfo.cpp

```diff
@@ -2432,10 +2432,13 @@ bool SIRegisterInfo::eliminateFrameIndex(MachineBasicBlock::iterator MI,
         if (Offset == 0) {
           unsigned OpCode = IsSALU && !LiveSCC ? AMDGPU::S_LSHR_B32
                                                : AMDGPU::V_LSHRREV_B32_e64;
-          // XXX - This never happens because of emergency scavenging slot at 0?
```

Comment:
Is this reachable now?

Reply:
Yep, I hit this in an OpenMP test after a [change to FixSGPRCopies](https://github.com/llvm/llvm-project/pull/66882).